package.json

```diff
@@ -4,21 +4,16 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "browser-install": "npx playwright install chromium",
-    "deps-install": "npx playwright install-deps",
+    "browser-install": "npx playwright install-deps && npx playwright install chromium",
     "test": "npx playwright test",
     "test-ui": "npx playwright test --ui",
-    "show-report": "npx playwright show-report",
-    "live-server": "npx live-server --port=5551"
```

Comment:
In case we get rid of the "live-server" script the readme.md first section should be updated too.